### PR TITLE
NetworkMessage methods

### PR DESCRIPTION
**Pull request by Mkalo**
I think this is very useful.

networkMessage:seek(position) -- sets the position
networkMessage:tell() -- return the current position
networkMessage:len() -- return the length of the buffer

Example of usage:
http://pastebin.com/zGyexExr

### CODE REVIEW

**Comment by ranisalt** [comment]
You should use `len()` instead of `length()` since Lua uses `len()` for everything.

File: src/luascript.cpp [hunk 1]
```diff
@@ -1910,6 +1910,9 @@ void LuaScriptInterface::registerFunctions()
 	registerMethod("NetworkMessage", "addItemId", LuaScriptInterface::luaNetworkMessageAddItemId);
 
 	registerMethod("NetworkMessage", "reset", LuaScriptInterface::luaNetworkMessageReset);
+	registerMethod("NetworkMessage", "seek", LuaScriptInterface::luaNetworkMessageSeek);
+	registerMethod("NetworkMessage", "tell", LuaScriptInterface::luaNetworkMessageTell);
+	registerMethod("NetworkMessage", "length", LuaScriptInterface::luaNetworkMessageLength);
```
Comment (djarek) [inline src/luascript.cpp]: Change the length to len

**Comment by DSpeichert** [comment]
@djarek Why is this scheduled for 2.0? Anything in this we can't have now?

**Comment by DSpeichert** [comment]
@Mkalo can you rebase this and maybe fix whatever errors are then thrown by CI?

**Pull request closed without merging**